pkg/endpoint/endpoint.go

```diff
@@ -290,6 +291,8 @@ type Endpoint struct {
 
 	realizedPolicy *policy.EndpointPolicy
 
+	*eventqueue.EventQueue
```

Comment:
Any particular reason not to make this a member? I was struggling a bit to understand `ep.Run()` and `ep.Stop()`. It contains almost no context.

Reply:
@joestringer suggested this in his review - it allowed me to remove boilerplate around starting and stopping the Endpoint's eventqueue (which I wanted to make an unexported field, but allow callers to access its lifecycle commands, e.g. `Run`, `Stop`, etc. There previously was a file within the endpoint package which I added which had wrappers around `eventqueue.Run`, etc., but now that is gone by using composition). I can make it a public field instead if you like, I'm not opinionated on this matter.

Reply:
Named the field `EventQueue` to avoid this confusion.

Reply:
I guess if it's `ep.Run()` then it might be easier to understand if it's instead either `ep.RunEventQueue` or `ep.EventQueue.Run()`.

Reply:
It's clear now, thanks @ianvernon